docs/opc-publisher/commandline.md

```diff
@@ -329,7 +329,7 @@ Transport settings
                                side car connection if needed.
                                Use the format 'PubSubComponent=<PubSubComponent>
                                [;GrpcPort=<GrpcPort>;HttpPort=<HttpPort>[;
-                               Scheme=<'https'|'http'>]][;ApiKey=<ApiKey>]'.
+                               Scheme=<'https'|'http'>][;Host=<Host>]][;ApiKey=<ApiKey>]'.
```

Comment:
Could you also update the CLI OPT help entry in commandline.cs?

Reply:
Done.